src/client/app/components/Graph.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './styling/Graph.css';
+
+class Graph extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    }
+  }
+
+  render() {
+    return (
+      <div className={styles.main}>
+        <ul className={styles.list}>
+          <li className={styles.entry}><div className={styles.rating}>Excellent</div><span className={styles.bar}><span style={{width: `${this.props.percentage[5]}%`}} className={styles.innerBar}></span></span><span>{this.props.rating[5]}</span></li>
```

Comment:
This is a tech debt thing, but you could set this up so that you map over an array and render a single item per element. Not a big deal though.